MediaBrowser.Providers/MediaBrowser.Providers.csproj

```diff
@@ -24,4 +24,9 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="Plugins\MusicBrainz\Configuration\config.html" />
```

Comment:
what this does?

Reply:
Common style in all plugins to remove the web assets from IDEs and such. The line right below adds it as an embedded resource in the final build.